examples/gnrc_border_router/Makefile

```
@@ -27,6 +27,19 @@ USEMODULE += ps
 # Optionally use DHCPv6 instead of UHCP
 USE_DHCPV6 ?= 0
 
+# Default to using ethos to provide the uplink when not on native
+USE_ETHOS ?= 1
+# Optionally use SLIP instead of ethos when not on native
+# (added to USEMODULE in Makefile.board.dep)
+USE_SLIP  ?= 0
+# Optionally use esp_wifi & esp_now with an esp8266 or esp32
+USE_ESP   ?= 0
+
+# When using a WiFi uplink we should use DHCPv6
+ifeq (1,$(USE_ESP))
+  USE_DHCPV6 = 1
```

```suggestion
  USE_DHCPV6 = 1
else
  USE_DHCPV6 ?= 0
endif
```

And remove what is currently line 28 above.